tests/console/zypper_in.pm

```diff
@@ -32,6 +45,20 @@ sub run {
     clear_console;    # clear screen to see that second update does not do any more
     assert_script_run("rpm -e $pkgname");
     assert_script_run("! rpm -q $pkgname");
+
+    script_run 'touch /preinstall_fail';
```

Comment:
Shouldn't some of those `script_run` have their exit status checked? Otherwise all of them might fail or get mistyped and the module wouldn't care at all.